channels/auth.py

```diff
@@ -20,7 +23,7 @@
 
 
 @database_sync_to_async
-def get_user(scope):
+def get_user(scope: Dict[str, Any]) -> Awaitable[Union[AbstractBaseUser, AnonymousUser]]:
```

Comment:
Is this correct? You don't **have** to inherit from `AbstractBaseUser` — it's just "...the easiest way..."

Reply:
It should, I was talking about it, before with @matthiask 

As a matter of fact the django docs say that for custom user you should inherit from AbstarctBaseUser. But yes it's the easier way to create a CustomUser, maybe we can add a new type for the user that deals with the settings and fetches the auth_user_model class? but it seems to much work for it

Reply:
It's not strictly required to inherit from `AbstractBaseUser`, that's true. But you're really on your own if you don't do this and you should absolutely know what you're doing then. And maybe you don't really need type *hints* either in this case.

Reply:
Fine. Emphasis on the “hints”.

Reply:
Since I don't like the idea to close the type to AbstractBaseUser, I think I will not assign any type, a simple Any is better since it can include all the cases